params/config.go

```diff
@@ -581,6 +582,13 @@ func (s ScrollConfig) String() string {
 		s.UseZktrie, maxTxPerBlock, maxTxPayloadBytesPerBlock, s.FeeVaultAddress, s.L1Config.String())
 }
 
+func (s ScrollConfig) GetMaxTxPerBlock() int {
```

Comment:
Nitpick but it might make more sense to put this into a miner cli parameter. Other fields in `ScrollConfig` are all values that fundamentally change or limit the execution (state machine, L1 data source, L1 limits, etc.), while this is just an optimization.

Reply:
Makes sense, will do.

Reply:
done